demo/index.html

```diff
@@ -1,53 +1,110 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <title>Slate Plugin</title>
-    <script src="https://unpkg.com/@playkit-js/kaltura-player-js@canary/dist/kaltura-ovp-player.js"></script>
-    <script src="https://unpkg.com/@playkit-js/audio-player@canary/dist/playkit-audio-player.js"></script>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta name="description" content="Kaltura Player Demo Page" />
+    <title>Mini Audio Player Plugin</title>
+    <link rel="stylesheet" type="text/css" href="./style.css" />
+    <!--    <script src="https://unpkg.com/@playkit-js/kaltura-player-js@canary/dist/kaltura-ovp-player.js"></script>-->
+    <!--    <script src="https://unpkg.com/@playkit-js/playkit-js-downloads@canary/dist/playkit-downloads.js" type="text/javascript"></script>-->
+    <!--    <script src="https://unpkg.com/@playkit-js/share@canary/dist/playkit-share.js" type="text/javascript"></script>-->
+    <script src="http://localhost:8086/kaltura-ovp-player.js"></script>
```

Comment:
links for localhost that will not work on other devices

Reply:
fixed